schema/project-level/project-schema.json

```
@@ -1346,6 +1346,12 @@
           "description": "The value of the budget line item.",
           "$ref": "#/definitions/Value"
         },
+        "approvalDate": {
+          "title": "Approval date",
+          "description": "The date on which this budget entry was approved. Where documentary evidence for this exists, it may be included among the project documents with [documentType](https://standard.open-contracting.org/infrastructure/latest/en/reference/codelists/#documenttype) set to 'budgetApproval'.",
```

Per the proposal in the issue, reference the field, not the codelist:

```suggestion
          "description": "The date on which this budget entry was approved. Where documentary evidence for this exists, it may be included among the project documents with `.documentType` set to 'budgetApproval'.",
```